### ISSUE TITLE
Suggestion: Auto disable on Hypixel/other servers

### ISSUE THREAD

**Issue by notrohit1702** [Celibistrial/freelook#19]
A feature that automatically disables the mod on Hypixel by default and a configurable (In mod menu) list of other servers would be great!

**Comment by notrohit1702** [comment]
I think this exists on some popular clients like Badlion and Lunar too.

**Comment by Celibistrial** [comment]
This should be easily addable, will do it when i get time

**Comment by pastadudes** [comment]
wait so freelook is bannable on hypixel?

**Comment by notrohit1702** [comment]
> wait so freelook is bannable on hypixel?

Yes, I don't know if it's bannable or just a kick but better be safe than sorry https://hypixel.net/threads/explaining-freelook-and-possibly-other-blacklisted-mods.4489770/

**Comment by notrohit1702** [comment]
> This should be easily addable, will do it when i get time

Hey, not to be a bother but, but is this still planned?
